common/scripted_triggers/wc_scripted_plot_triggers.txt

```diff
@@ -0,0 +1,76 @@
+can_be_killed_by_poisoned_wine_trigger = {
+	can_be_drunkard_trigger = yes
+}
+can_be_killed_by_falling_trigger = {
+	is_humanoid_trigger = yes
+	dark_public_religion_trigger = no
+	
+	can_fly_trigger = no
+}
+can_be_killed_by_archer_ambush_trigger = {
+	#Tribal/nomadic ruler touring in a carriage... sounds weird... don't know why Paradox haven't disabled it
+	is_tribal = no
+	is_nomadic  no
+	
+	is_humanoid_trigger = yes
+	dark_public_religion_trigger = no
+}
+can_be_killed_by_manure_explosion_trigger = {
+	#Tribal/nomadic ruler touring and entering taverns... sounds weird... don't know why Paradox haven't disabled it
```

Comment:
What about tribal monarchy?

Reply:
Well, it says nothing about a carriage and most likely they have some kind of taverns, so why not? Allowed in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/pull/1219/commits/0d514b9deee420a8667987c748a9ca7512663f8c.